lib/state/appActions.js

```diff
@@ -127,8 +160,8 @@ appActions.load = function(cb) {
     setHostsWithCallback,
   ], function(err, results) {
     if (err) {
-      var error = new UploaderError(err.message, appActions.errorStage.STAGE_SETUP, err);
-      return cb(error);
+      appActions.addMoreInfoToError(err, 'STAGE_SETUP');
```

Comment:
I think we  should just just pass in the actual stage 
 
``appActions.addMoreInfoToError(err, appActions.errorStage.STAGE_SETUP);``